source/_theme/pydy/layout.html

```diff
@@ -1,10 +1,10 @@
 {% extends "basic/layout.html" %}
 
-{% set theme_css_files = ['_static/main.css', '_static/custom.css',
+{% set theme_css_files = ['/_static/main.css', '/_static/custom.css',
```

Comment:
I don't agree with this change. Can you prove to me why it is correct?

I've built numerious Sphinx sites and plenty of of static sites without having to do this.

I don't think you should be required to view locally with a server.